common/production_methods/imperia_industry.txt

```diff
@@ -22,3 +21,134 @@
 		}
 	}
 }
+
+pm_basic_charcoal_production = {
+	texture = "gfx/interface/icons/production_method_icons/picks_and_shovels.dds"
+	building_modifiers = {
+		workforce_scaled = {
+			# input goods
+			goods_input_wood_add = 15
+			# output goods
+			goods_output_coal_add = 30
+			#Productivity with merchant guilds: 6.24 -> ~8% more efficient than basic pm coal mine
+		}
+		level_scaled = {
+			building_employment_laborers_add = 4500
+		}
+	}
+}
+
+pm_improved_charcoal_production = {
+	texture = "gfx/interface/icons/production_method_icons/picks_and_shovels.dds"
```

Comment:
needs proper icon

Reply:
![charcoal_icons](https://github.com/Imperia-Development-Victoria-3/Victoria-3-Dev/assets/44327039/22eb297b-0386-4047-b497-7acadd196090)
Had some trouble finding suitable icons, but these should evocate the feeling well enough.